.github/workflows/covector-version-or-publish.yml

```diff
@@ -33,13 +33,17 @@ jobs:
           CARGO_AUDIT_OPTIONS: ${{ secrets.CARGO_AUDIT_OPTIONS }}
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          command: 'version-or-publish'
+          command: "version-or-publish"
           createRelease: true
           recognizeContributors: true
 
+      - name: Sync Cargo.lock
+        if: steps.covector.outputs.commandRan == 'version'
+        run: cargo tree --depth 0
```

Comment:
This probably needs to be in covector config instead

Reply:
i don't touch covector stuff 😅 

copied that from the tauri workflows btw, no idea who put that there